docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -55,93 +57,92 @@ The ``uw`` mode for handling configs.
     --verbose, -v
           Print all logging messages
 
-
 Examples
-~~~~~~~~
+^^^^^^^^
 
-The examples that follow use namelist files ``values.nml`` and ``values2.nml`` with content
+The examples that follow use namelist files ``values1.nml`` and ``values2.nml``, both initially with content
```

Comment:
Here and below, when there are two files with similar purpose, I've named them with an appended `1` and `2`, respectively, instead of having only the second one have a `2` appended.